src/jobs/time-tracking-job.js

```diff
@@ -11,30 +11,48 @@
  *
  */
 
+const moment = require('moment');
+
 const BaseJobHandler = require('./base/base-job-handler');
+const FogAccessTokenService = require('../services/iofog-access-token-service');
+const logger = require('../logger');
 const Tracking = require('../tracking');
 const TrackingEventType = require('../enums/tracking-event-type');
+const TransactionDecorator = require('../decorators/transaction-decorator');
+
+
+const INTERVAL = 5 * 60 * 1000;
 
 class TimeTrackingJob extends BaseJobHandler {
 
   constructor() {
     super();
-    this.scheduleTime = intervalMin * 60 * 1000;
+    this.startTime = moment.now();
   }
 
   run() {
-    setInterval(trackTime, this.scheduleTime);
+    setTimeout(this.trackTime, INTERVAL);
```

Comment:
why you prefer setTimeout() instead of setInterval()? 
I think that setInterval() more accurate because it's interval always be the same and equal to constant provided in args. But for setTimeout() real interval will be constant + time of trackTime function execution (which can be different each time)

Reply:
`setInterval` is called `EVIL` in JS, there are tons of articles about it. The execution interval is not necessarily important in our case, so sending events with few milliseconds delay is not a big deal.